mimoto-issuers-config.json

```diff
@@ -220,13 +220,93 @@
+            "alt_text": "شعار مربع لطائر الشمس"
+          },
+          "language": "ar",
+          "title": "قم بتنزيل بيانات اعتماد Sunbird",
+          "description": "قم بتنزيل بيانات الاعتماد باستخدام رقم السياسة الخاص بك"
+        },
+        {
+          "name": "सनबर्ड बीमा",
+          "logo": {
+            "url": "https://sunbird.org/images/sunbird-logo-new.png",
+            "alt_text": "सनबर्ड का एक चौकोर लोगो"
+          },
+          "language": "hi",
+          "title": "सनबर्ड क्रेडेंशियल्स डाउनलोड करेंं",
+          "description": "अपने पॉलिसी नंबर के साथ क्रेडेंशियल डाउनलोड करें"
+        },
+        {
+          "name": "ಸನ್ ಬರ್ಡ್",
+          "logo": {
+            "url": "https://sunbird.org/images/sunbird-logo-new.png",
+            "alt_text": "ಸನ್ ಬರ್ಡ್ ನ ಚೌಕಾಕಾರದ ಲೋಗೋ"
+          },
+          "language": "kn",
+          "title": "ಸನ್ಬರ್ಡ್ ರುಜುವಾತುಗಳನ್ನು ಡೌನ್ಲೋಡ್ ಮಾಡಿ",
+          "description": "ನಿಮ್ಮ ಪಾಲಿಸಿ ಸಂಖ್ಯೆಯೊಂದಿಗೆ ರುಜುವಾತುಗಳನ್ನು ಡೌನ್ಲೋಡ್ ಮಾಡಿ"
+        },
+        {
+          "name": "சன்பேர்ட் காப்பீடு",
+          "logo": {
+            "url": "https://sunbird.org/images/sunbird-logo-new.png",
+            "alt_text": "சூரிய பறவையின் சதுர சின்னம்"
+          },
+          "language": "ta",
+          "title": "Sunbird சான்றுகளைப் பதிவிறக்கவும்",
+          "description": "உங்கள் பாலிசி எண்ணுடன் நற்சான்றிதழ்களைப் பதிவிறக்கவும்"
+        },
+        {
+          "name": "Seguro sa Sunbird",
+          "logo": {
+            "url": "https://sunbird.org/images/sunbird-logo-new.png",
+            "alt_text": "isang parisukat na logo ng isang Sunbird"
+          },
+          "language": "fil",
+          "title": "I-download ang Mga Kredensyal ng Sunbird",
+          "description": "Mag-download ng mga kredensyal gamit ang iyong Numero ng Patakaran"
+        }
+      ],
+      "protocol": "OpenId4VCI",
+      "client_id": "${mimoto.oidc.sunbird.partner.clientid}",
+      "client_alias": "esignet-sunbird-partner",
+      "scopes_supported": ["sunbird_rc_insurance_vc_ldp"],
+      "additional_headers": { "Accept": "application/json" },
+      ".well-known": "https://${mosip.esignet.insurance.host}/.well-known/openid-credential-issuer?version=v11",
+      "redirect_uri": "io.mosip.residentapp.inji://oauthredirect",
+      "authorization_endpoint": "https://${mosip.esignet.insurance.host}/authorize",
+      "authorization_audience": "https://${mosip.esignet.insurance.host}/v1/esignet/oauth/v2/token",
+      "token_endpoint": "https://${mosip.api.public.host}/residentmobileapp/get-token/Sunbird",
+      "proxy_token_endpoint": "https://${mosip.esignet.insurance.host}/v1/esignet/oauth/v2/token",
+      "credential_endpoint": "https://${mosip.esignet.insurance.host}/v1/esignet/vci/credential",
+      "credential_type": ["VerifiableCredential", "InsuranceCredential"],
+      "credential_audience": "https://${mosip.esignet.insurance.host}",
+      "enabled": "false"
+    },
```

Comment:
Why we need this change ? Veridonia is a insurance use case. sunbird is not a issuer, just a background technology used.

Reply:
Veridonia use case added for Id4Africa, otherwise it was Sunbird as an issuer. So we thought to disable the Sunbird and add Veridonia as an new issuer.

Reply:
Sunbird is one of our existing issuer, which we renamed to "veridonia". we kept it, if in case its needed for existing collab users, we can enable it. if we are expecting this veridonia to be displayed here after. removing it.